src/marks/cell.js

```diff
@@ -54,7 +55,7 @@ export function cell(data, options = {}) {
  */
 export function cellX(data, options = {}) {
   let {x = indexOf, fill, stroke, ...remainingOptions} = options;
-  if (fill === undefined && maybeColorChannel(stroke)[0] === undefined) fill = identity;
+  if (fill === undefined && definition("stroke", stroke)[0] === undefined) fill = identity;
```

Comment:
Maybe maybeChannel would work as a more specific name here?

```suggestion
  if (fill === undefined && maybeChannel("stroke", stroke)[0] === undefined) fill = identity;
```